Ginger/Unit Tests/NonUITests/UsedVariablesTest.cs

```diff
@@ -138,7 +138,7 @@ public void UsedVarInActionOutoutStoreToVar()
             //Arrange
             Activity a1 = new Activity() { ActivityName = "a1" };
             ActDummy act1 = new ActDummy();
-            ActReturnValue actReturnValue = new ActReturnValue() { Param = "out1", StoreToVariable = "v1" };
+            ActReturnValue actReturnValue = new ActReturnValue() { Param = "out1", StoreToValue = "v1" };
```

Comment:
need to keep store to variable - why changed to value?

Reply:
we have removed StoreToVariable field , thats why cannot keep as StoreToVariable .